ci/test-coverage.sh

```diff
@@ -29,6 +29,12 @@ scripts/coverage.sh
 report=coverage-"${CI_COMMIT:0:9}".tar.gz
 mv target/cov/report.tar.gz "$report"
 upload-ci-artifact "$report"
+
+stderr_log=target/cov/coverage-stderr.log
+compressed_stderr_log="${stderr_log}.gz"
+gzip < "$stderr_log" > "$compressed_stderr_log"
+upload-ci-artifact "$compressed_stderr_log"
```

Comment:
How about this instead:
```bash
gzip target/cov/coverage-stderr.log
upload-ci-artifact target/cov/coverage-stderr.log.gz
```

Reply:
yeah, It looks nicer; https://github.com/solana-labs/solana/pull/6345/commits/ccafb52d5a57d24b388662c306aa8160c76d4bbe.

Just that I'm too fond of redirections and pipes.. :p